### ISSUE TITLE
Update model.fit docs to indicate what the default params are

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#1846]
https://github.com/tensorflow/tfjs-examples/blob/master/webcam-transfer-learning/index.js says

// Train the model! Model.fit() will **shuffle** xs & ys so we don't have to.
  ```
  model.fit(controllerDataset.xs, controllerDataset.ys, {
    batchSize,
    epochs: ui.getEpochs(),
    callbacks: {
      onBatchEnd: async (batch, logs) => {
        ui.trainStatus('Loss: ' + logs.loss.toFixed(5));
      }
    }
  });
}
```
While the documentation for fit says only the following regarding shuffling:

> shuffle (boolean) Whether to shuffle the training data before each epoch. Has no effect when stepsPerEpoch is not null.

Unless "shuffle" is true by default (which should be well-documented if somehow true) then comment on example code is incorrect.

**Comment by bileschi** [comment]
Agreed, this should be clearer:
https://github.com/tensorflow/tfjs/blob/master/tfjs-layers/src/engine/training_tensors.ts#L287

**Comment by tafsiri** [comment]
I'm going to re-open this and rename it to updating the docs for fit to indicate what the default parameters are for model.fit.

What do you think @bileschi @caisq? Would you be able to take this on?

**Comment by caisq** [comment]
`shuffle` is `true`by default for `tf.LayersModel.fit()`, which is consistent with Python Keras

See: https://github.com/tensorflow/tfjs/blob/master/tfjs-layers/src/engine/training_tensors.ts#L285

I'll update the doc string.

**Comment by dhirensr** [comment]
@rthadur @tafsiri : please review PR for this issue https://github.com/tensorflow/tfjs/pull/2869 . :smile:

**Comment by rthadur** [comment]
Related PR has been merged. Closing this Issue.
